src/main.rs

```diff
@@ -54,3 +77,59 @@ fn main() {
     let scores = sample_aggregate_scores::<5, rand::rngs::ThreadRng>(&mut rng, 20_000);
     println!("{scores}")
 }
+
+pub fn five_card(samples_number: u32){
+    let mut rng = rand::thread_rng();
+
+    println!("Sorted Deck:");
+    for id in 0..52 {
+        let card = Card { id };
+        if card.suit().id == 3 {
+            println!("  {}", card);
+        } else {
+            print!("  {}  ", card);
+        }
+    }
+
+    println!("");
+    let five_card_hand = Hand::<5>::draw(&mut rng);
+    println!("{five_card_hand}");
+    let hand_stats = HandStats::from(&five_card_hand);
+    println!("{hand_stats}");
+    let hand_score = HandScore::from(&hand_stats);
+    println!("{hand_score}");
+
+    // Now draw N random hands and check the stats!
+    println!("");
+    // Note: there is probably some way to deduce the type of the RNG here...
+    let scores = sample_aggregate_scores::<5, rand::rngs::ThreadRng>(&mut rng, samples_number);
+    println!("{scores}")
+}
+
+pub fn seven_card(samples_number: u32){
```

Comment:
It looks like this function is a near duplicate of `five_card()` -- there should instead be a single common function with a parameter for the number of cards. There is also no reason that 5 and 7 are particularly special. Any positive integer would be fine... although you won't be able to get some hands. (I think). Also, the name could be more descriptive.

Reply:
can I remove those constants from the hand because if  I use one function to handle all of them I need to pass non constant value there so either I can solve it by using if else statement or it's needed to change constant to non constant in hand.rs

Reply:
Yeah - this one is slightly trickier than I had first thought because the hand size is a compile time parameter. For now, I think it is fine to only support 5 and 7 card hands. We can add more later if needed. You can take a look at some of the library code to see how to generalize over compile time parameters.